src/components/Login/LoginForm.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Form } from "react-final-form";
+import { useTranslation } from "react-i18next";
+
+import {
+  emailRegex,
+  required,
+  composeValidators,
+} from "../../utils/validators";
+
+import EmailField from "../UI/EmailField";
+import PasswordField from "../UI/PasswordField";
+import Action from "../UI/Action";
+
+const LoginForm = (props) => {
+  const { t } = useTranslation();
+  return (
+    <Form
+      onSubmit={props.handleSubmit}
+      render={({ submitError, modifiedSinceLastSubmit, handleSubmit }) => (
+        <div className="flex justify-center mr-8">
+          <form onSubmit={handleSubmit} className="w-full max-w-sm">
+            <EmailField
+              name="email"
+              placeholder={t("description.emailPlaceholder")}
+              labelText={t("description.email")}
+              validate={composeValidators(required, emailRegex)}
+            />
+            <PasswordField
+              name="password"
+              placeholder={t("description.passwordPlaceholder")}
+              labelText={t("description.password")}
+              validate={composeValidators(required)}
```

Comment:
Component can do that itself, just tell it if it is required. So you don't have to import `composeValidators` and `required` in parent form. It's not its business.